models/twilio.yml

```diff
@@ -39,7 +39,7 @@ models:
         description: Total number of read messages for this phone number. 
       - name: total_messages
         description: Total number of messages for this phone number. 
-      - name: total_spent
+      - name: total_spend
```

Comment:
I wanted to rename these fields to be `spend` as it was more reflective of the metric.